app/src/main/res/values/strings.xml

```diff
@@ -6,6 +6,9 @@
     <string name="app_name_private_5">Private %s</string>
     <!-- App name for private browsing mode. The first parameter is the name of the app defined in app_name (for example: Fenix)-->
     <string name="app_name_private_4">%s (Private)</string>
+    <string name="gdpr_has_you_covered_firefox_has_gdpr_covered">%s (Private)</string>
+    <string name="sharing_your_phone_autoclosing_private_tabs_is_for_you">%s (Private)</string>
+    <string name="love_firefox_fill_in_our_survey">%s (Private)</string>
```

Comment:
@jhugman To be able of compiling, I had to add these mock strings. I was seeing those in the compiled code but I'm not quite sure from where they were fetched or how are we to add those strings.